.rubocop.yml

```diff
@@ -15,6 +15,11 @@ inherit_mode:
   merge:
     - Exclude
 
+Layout/LineLength:
+  Enabled: true
+  Exclude:
+    - config/application.rb
```

Comment:
Is there a reason this cop has been enabled?

This neither an endorsement or disapproval, but merely a question 💭

Reply:
its enabled by default out of govuk rubocop or rubocop itself.

Reply:
i can remove the enabled from here actually actually as it is already enabled

Reply:
actually its prettier which enables it